src/ceph-volume/ceph_volume/devices/lvm/deactivate.py

```diff
@@ -0,0 +1,96 @@
+import argparse
+import logging
+import sys
+from textwrap import dedent
+from ceph_volume import conf
+from ceph_volume.util import encryption, system
+from ceph_volume.api.lvm import get_lvs_by_tag
+
+logger = logging.getLogger(__name__)
+
+
+def deactivate_osd(osd_id=None, osd_uuid=None):
+
+    lvs = []
+    if osd_uuid:
+        lvs = get_lvs_by_tag('ceph.osd_fsid={}'.format(osd_uuid))
```

Comment:
```suggestion
        lvs = get_lvs_by_tag('ceph.osd_uuid={}'.format(osd_uuid))
```

I thought we wanted to move away from fsid?

Reply:
yup but the `ceph.osd_fsid` tag is what's out there today.

Reply:
Changing fsid to uuid is out of scope for these changes, and I don't think ceph-volume should try to use one vs. the other unless the rest of the Ceph project decides to do so.

Here is just one of the many places where fsid is used https://docs.ceph.com/docs/master/man/8/ceph-osd/

Keeping the scope of this PR just to address the problem at hand should be preferred.